services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -43,8 +49,20 @@ export const AppRoutes = () => {
           ))}
           <Route path="/mcpar/export" element={<ExportedReportPage />} />
           <Route path="/mcpar/*" element={<Navigate to="/mcpar" />} />
-          <Route path="/profile" element={<ProfilePage />} />
-          <Route path="*" element={<NotFoundPage />} />
+
+          {/* MLR ROUTES */}
+          <Route path="/mlr" element={<DashboardPage reportType="MLR" />} />
+          <Route
+            path="/mlr/get-started"
+            element={<ReportGetStartedPage reportType="MLR" />}
+          />
+          {mlrReportJson.flatRoutes.map((route: ReportRoute) => (
+            <Route
+              key={route.path}
+              path={route.path}
+              element={<ReportPageWrapper />}
+            />
+          ))}
```

Comment:
Question: How do we want to block these routes if the MLR feature flag is not enabled? Not render the routes? Redirect on page load? Open to ideas.

Reply:
I'd vote for a redirect on page load, probably. If someone types in a path we can just redirect them to the homepage.